internal/tools/start.go

```diff
@@ -60,14 +74,41 @@ func handlerStartPipeline(ctx context.Context, request mcp.CallToolRequest) (*mc
 			PipelineRef: &v1.PipelineRef{
 				Name: name,
 			},
+			Params: pipelineParams,
 		},
 	}
 
-	if _, err := pipelineclientset.TektonV1().PipelineRuns(namespace).Create(ctx, pr, metav1.CreateOptions{}); err != nil {
+	createdPr, err := pipelineclientset.TektonV1().PipelineRuns(namespace).Create(ctx, pr, metav1.CreateOptions{})
+	if err != nil {
 		return mcp.NewToolResultError(fmt.Sprintf("Failed to create PipelineRun %s/%s: %v", namespace, name, err)), nil
 	}
 
-	return result(fmt.Sprintf("Starting pipeline %s in namespace %s", name, namespace)), nil
+	// Include parameter information in the result message
+	resultMsg := fmt.Sprintf("Starting pipeline %s in namespace %s", name, namespace)
+	if len(pipelineParams) > 0 {
+		paramDescs := make([]string, 0, len(pipelineParams))
+		for _, p := range pipelineParams {
+			var valueStr string
+			switch p.Value.Type {
```

Comment:
This switch statement is the same one as in handlerStartTask function. Can it extract to a standalone function?